packages/connect-popup/webpack/prod.webpack.config.ts

```diff
@@ -11,29 +11,47 @@ export default {
     target: 'web',
     mode: 'production',
     entry: {
-        popup: path.resolve(__dirname, '../src/index.ts'),
+        popup: path.resolve(__dirname, '../src/index.tsx'),
     },
     output: {
         filename: 'js/[name].[contenthash].js',
         path: DIST,
         publicPath: './',
     },
+    // todo: not 100% sure about this config. lets focus on this in review
```

Comment:
seems good enough but in future we should create some kind of shared webpack base config that will be used in all of other webpack configs because lot of thigs here are just copy pasted